elasticsearch/install.sh

```diff
@@ -8,19 +8,27 @@ source ${HOME}/prep-install.${RELEASE_STREAM}
 echo "ES plugins: ${es_plugins[@]}"
 for es_plugin in ${es_plugins[@]}
 do
-  ${ES_HOME}/bin/elasticsearch-plugin install -b $es_plugin
+  if [ -x ${ES_HOME}/bin/elasticsearch-plugin ] ; then
+    plugincmd=${ES_HOME}/bin/elasticsearch-plugin
+  else
+    plugincmd=${ES_HOME}/bin/plugin
+  fi
+  $plugincmd install -b $es_plugin
 done
 
-
 #fix location from config
 if [[ "${ES_HOME}" != "/usr/share/elasticsearch" ]]; then
   ln -s ${ES_HOME}/index_templates /usr/share/elasticsearch/index_templates
   ln -s ${ES_HOME}/index_patterns /usr/share/elasticsearch/index_patterns
   ln -s ${ES_HOME}/kibana_ui_objects /usr/share/elasticsearch/kibana_ui_objects
 fi
 
-mkdir /elasticsearch
-mkdir -p $ES_CONF
+if [ ! -d /elasticsearch ] ; then
```

Comment:
I don't know if its worth it here? but we should consider moving this to a place that is modifiable when we have to run on atomic/coreos.  That would be something like `/var/lib/elasticsearch` right?

Reply:
Does it really matter if this is a volume mount anyway?

Reply:
It only matters if it matters to how it interacts with atomic/coreos.  If mounted volumes are allowed to be in this location the its a non-issue.

Reply:
I think mounted volumes are allowed in almost every location